.github/workflows/core.yml

```diff
@@ -65,13 +65,13 @@ jobs:
             runner-image: ubuntu-20.04
           - os: win64
             runner-image: windows-2022
-    steps:
+    steps:  
       - uses: IDAES/idaes-ui/.github/actions/install@main
         with:
           variant: ${{ matrix.install-variant }}
           python-version: ${{ matrix.python-version }}
       - name: Run pytest
-        run: pytest -v --pyargs idaes_ui.fv
```

Comment:
@CopyDemon this needs to be reverted for the CI job to run correctly on both `dev` and `standard` install variants. In other words, the `--pyargs idaes_ui.fv` should be added back.

Reply:
Thanks for the comment and it's helps a lot. Sorry for goes back and forth on this one several times.

Reply:
No worries, the current CI setup is functional but not super intuitive so I'm more than happy to help whenever there are issues.